benchmarks/libxslt_xpath/Dockerfile

```diff
@@ -58,12 +58,9 @@ RUN apt-get update && \
 
 
 RUN git clone \
-        --branch v2.10.3 \
-        --depth 1 \
         https://gitlab.gnome.org/GNOME/libxml2.git && \
+    git -C libxml2 checkout c7260a47f19e01f4f663b6a56fbdc2dafd8a6e7e
```

Comment:
It's weird we are going to checkout one commit here and then another in benchmark.yaml. Can we do without the checkout here?

Reply:
IIUC, `libxslt` is the benchmark target and `libxml2` is its dependency?
This commit ID is for `libxml2` and the commit ID in `benchmark.yaml` is for `libxslt`.

Reply:
Ah my bad.